File: programmers/js/level01/12918.js
// 문자열 다루기 기본
function solution(s){
  return parseInt(s) === Number(s) ? 
    (s.length === 4 || s.length === 6) ? true : false :
    false;
}

// 다른 풀이
// function solution(s){
//   var regex = /^\d{6}$|^\d{4}$/;

//   return regex.test(s);
// }

File: programmers/java/level01/12918.java
// 문자열 다루기 기본

class Solution {
    boolean isNum(String str) {
        try {
            Integer.parseInt(str);
            return true;
        } catch (NumberFormatException error) {
            return false;
        }
    }
    
    public boolean solution(String s) {
        return (s.length() == 4 || s.length() == 6 )
            && isNum(s);
    }
}